classroom.rb

```diff
@@ -0,0 +1,14 @@
+class Classroom
+  attr_accessor :label
+  attr_reader :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    @students.push(student)
```

Comment:
- [ ] Let's modify this method so it serves as the 'has-many side by replacing  `@students.push(student)` to 
`@students << student`

Reply:
![Screen Shot 2021-10-07 at 11 13 31 AM](https://user-images.githubusercontent.com/63879921/136355723-fce6f82f-dc33-44b9-8f25-1a762d9620d6.png)